src/app/(cities)/lodz/page.mdx

```diff
@@ -1,5 +1,60 @@
+import {env} from '../../../env';
+import {CityBanner} from '../../../components/CityBanner';
+import {LocalGroups} from "../../../components/LocalGroups";
+import {CityEmail} from "../../../components/CityEmail";
+import {FAQ} from "../../../components/FAQ";
+import {Organizers} from "../../../components/Orginizers";
+import {LodzFAQ} from "./_FAQ";
+import {LodzOrganizers} from "./_Organizers";
+
 export const metadata = {
+    metadataBase: new URL(`${env.SITE_URL}/lodz`),
 	title: "meet.js Łódź",
+    description: 'JavaScript meetups in Łódź',
+    keywords: 'meet.js, Łódź, JavaScript, meetup',
+    openGraph: {
+        title: 'meet.js Łódź',
+        description: 'JavaScript meetups in Łódź',
+        type: 'website',
+        locale: 'en_US',
+        url: `${env.SITE_URL}/lodz`,
+        siteName: 'meet.js',
+        images: [
+            {
+                url: `${env.SITE_URL}/api/og?city=Lodz`,
+                width: 1200,
+                height: 630,
+                alt: 'meet.js | JavaScript meetups in Łódź',
+            },
+        ],
+    },
+    twitter: {
+        card: 'summary_large_image',
+        site: '@meetjs',
+        title: 'meet.js',
+        description: 'JavaScript meetups in Łódź',
+        images: [
+            {
+                url: `${env.SITE_URL}/api/og?city=Lodz`,
+                width: 1200,
+                height: 630,
+                alt: 'meet.js | JavaScript meetups in Łódź',
+            },
+        ],
+    },
 };
 
-# Łódź
\ No newline at end of file
+<CityBanner city="Łódź" background="city/lodz.jpg" />
+
+<LocalGroups
+    localGroups={[
+        'https://www.meetup.com/meet-js-lodz/',
+        'https://lu.ma/user/usr-eNzoaGXYjizOBQI',
```

Comment:
We can now add a link to lu.ma with a city filter https://lu.ma/meet.js?tag=%C5%82%C3%B3d%C5%BA but we don't have to (if you prefer)